core/src/calleeFunctions/helpers/uniswapV3.ts

```diff
@@ -66,3 +66,17 @@ export const convertCollateralToDai = async function (
     const daiAmount = new BigNumber(daiIntegerAmount._hex).shiftedBy(-DAI_NUMBER_OF_DIGITS);
     return daiAmount;
 };
+
+export const convertDaiToMkr = async function (network: string, amountDai: BigNumber): Promise<BigNumber> {
```

Comment:
Sorry, I think I misunderstood your question during one of the standups. Please refactor `convertCollateralToDai` into `convertCollateralToDai`+`convertSymbolToDai`. Where you then can provide `MKR`? otherwise there is too much duplicative code